README.md

```diff
@@ -10,27 +15,56 @@ git clone https://github.com/kind-lab/mimic-fhir.git
 ```sh
 psql -f create_fhir_tables.sql
 ```
+  - In order to confirm the tables were generated, go into postgres using the command `psql` and connect to the database for which you loaded all your data into (eg `\c mimiciv`). Once connected, check your schema's using the `\dn`to check if you have the `mimic-fhir` schema. Once confirmed, run the following command in the postgres terminal `SET SEARCH_PATH TO MIMIC-FHIR;` and run `\dt` to see if the relations were made.
```

Comment:
Do you think it would be best to remove this part as it is just to check if the previous steps were done correctly? If not, I will change the text to lowercase.

Reply:
No, I think validation is always useful. Better yet you could even write a validate script similar to what's done in the MIMIC build repo, where it asserts that the row count from a series of selects is `> 0`.

Whether you do the validation here (after table creation but before the rows are inserted), or later (after data have been inserted) is a good question. I would probably defer to doing it after the data have been inserted.